build.sbt

```diff
@@ -64,15 +64,18 @@ lazy val membership = project
   .in(file("membership"))
   .settings(stdSettings("zio-membership"))
   .settings(
+    mainClass in (Compile, run) := Some("zio.membership.Main"),
     libraryDependencies ++= Seq(
-      "dev.zio"                %% "zio"                     % "1.0.0-RC17",
-      "dev.zio"                %% "zio-streams"             % "1.0.0-RC17",
-      "dev.zio"                %% "zio-nio"                 % "0.4.0",
-      "dev.zio"                %% "zio-macros-core"         % "0.6.0",
-      "com.lihaoyi"            %% "upickle"                 % "0.8.0",
-      "org.scala-lang.modules" %% "scala-collection-compat" % "2.1.2",
-      "dev.zio"                %% "zio-test"                % "1.0.0-RC17" % Test,
-      "dev.zio"                %% "zio-test-sbt"            % "1.0.0-RC17" % Test
+      "dev.zio"     %% "zio"             % "1.0.0-RC17",
+      "dev.zio"     %% "zio-streams"     % "1.0.0-RC17",
+      "dev.zio"     %% "zio-nio"         % "0.4.0",
+      "dev.zio"     %% "zio-macros-core" % "0.6.0",
+      "com.lihaoyi" %% "upickle"         % "0.8.0",
+      "dev.zio"     %% "zio-test"        % "1.0.0-RC17" % Test,
+      "dev.zio"     %% "zio-test-sbt"    % "1.0.0-RC17" % Test,
+      ("com.github.ghik" % "silencer-lib" % "1.4.4" % Provided).cross(CrossVersion.full),
+      compilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.1"),
+      compilerPlugin(("com.github.ghik" % "silencer-plugin" % "1.4.4").cross(CrossVersion.full))
```

Comment:
This can go to `BuildHelper` as it's done in other projects in the organization. The same applies to `silencer-lib`.

Reply:
Will do this in a separate pr.